specification/mediaservices/resource-manager/Microsoft.Media/preview/2018-03-30-preview/StreamingPoliciesAndStreamingLocators.json

```diff
@@ -0,0 +1,1248 @@
+          "items": {
+            "$ref": "#/definitions/StreamingPolicy"
+          },
+          "description": "A collection of StreamingPolicy items."
+        },
+        "@odata.nextLink": {
+          "type": "string",
+          "description": "A link to the next page of the collection (when the collection contains too many results to return in one response)."
+        }
+      },
+      "type": "object",
+      "description": "A collection of StreamingPolicy items."
+    },
+    "StreamingLocatorCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StreamingLocator"
+          },
+          "description": "A collection of StreamingLocator items."
+        },
+        "@odata.nextLink": {
+          "type": "string",
+          "description": "A link to the next page of the collection (when the collection contains too many results to return in one response)."
+        }
+      },
+      "type": "object",
+      "description": "A collection of StreamingLocator items."
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/streamingPolicies": {
+      "get": {
+        "summary": "List Streaming Policies",
+        "description": "Lists the Streaming Policies in the account",
+        "operationId": "StreamingPolicies_List",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StreamingPolicyCollection"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Lists Streaming Policies": {
+            "$ref": "examples/streaming-policies-list.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/streamingPolicies/{streamingPolicyName}": {
+      "get": {
+        "summary": "Get a Streaming Policy",
+        "description": "Get the details of a Streaming Policy in the Media Services account",
+        "operationId": "StreamingPolicies_Get",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StreamingPolicy"
+            }
+          },
+          "404": {
+            "description": "NotFound"
+          },
```

Comment:
Unsuccessful status codes should not be included

Reply:
Generally, I agree but by including 404 the client will return a null for the Get operation as opposed to throwing an exception.  We are doing this consistently across the Azure Media Services SDK.